modules/ndu/src/backend/db.ts

```diff
@@ -0,0 +1,58 @@
+import * as sdk from 'botpress/sdk'
+import _ from 'lodash'
+
+export const rankingColumns = [
+  'topicName',
+  'topicConfidence',
+  'topicIntentName',
+  'topicIntentConfidence',
+  'topicIntentConfidenceAdjusted', // topicConfidence * topicIntentConfidence..formula need adjustment
+  'topicIntentType'
+]
+
+export const nbRankingIntent = 3
+
+export default class Database {
+  private knex: sdk.KnexExtended
+
+  constructor(bp: typeof sdk) {
+    this.knex = bp.database
+  }
+
+  async initialize() {
+    // The schema of this table is subject to change often while in development.
+    // await this.knex.schema.dropTable('ndu')
+
+    return this.knex.createTableIfNotExists('ndu', table => {
+      table.increments('id').primary()
+      table.string('incomingEventId')
+      table.string('currentTopicName')
```

Comment:
You do have a point, I guess I was just lazy and didn't want to implement an object mapper to convert between snake_case and camelCase. 

However, since the structure of that table is temporary and subject to change a couple of time, i'd like to keep the columns name the same as the object property, i'll fix it with snake case when we settle on the final structure